### DISCUSSION TITLE
Module not found: Error: Can't resolve '../build'

### DISCUSSION THREAD

**Discussion by droidwizard** [dolanmiu/docx#1939] [General]
I installed docx in last version **"docx": "^7.8.2"**
and import module into my project 
**import { Document, Paragraph, TextRun } from '../build'**
It show error message is **Module not found: Error: Can't resolve '../build'**
My project is Nuxtjs, Help me solve this issue., Thanks.

**Comment by dolanmiu** [comment]
Because you need to change `../build` to `docx`
